Library/Homebrew/cmd/help.rb

```diff
@@ -1,11 +1,18 @@
-# typed: strict
+# typed: strong
 # frozen_string_literal: true
 
+require "abstract_command"
 require "help"
 
 module Homebrew
-  sig { returns(T.noreturn) }
-  def help
-    Help.help
+  module Cmd
+    class HelpCmd < AbstractCommand
+      cmd_args { description "Print help information." }
```

Comment:
Is this really necessary? At a glance it seems like it would never get used.

Reply:
Eh, not really. I think I have some follow-up PRs in the queue, for when every command is an `AbstractCommand`, where it simplifies things a bit if every Command also has a `description`. But we can consider this at that time instead.

Reply:
(it was necessary, see https://github.com/Homebrew/brew/pull/17038)